### PR TITLE
feat(my account): add admin info to issuer info table

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

User Account & My User Account: Add Admin Info

## Why

Adding Admin Emails to Issuer Table

## Issue

CPLP-2815

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

**Review by lavanya-bmw — APPROVED** [review]
lgtm

File: cx-portal/src/features/admin/userOwn/mapper.ts [hunk 1]
```diff
@@ -18,7 +18,13 @@
...
  ********************************************************************************/
 
-import { UserDetail } from './types'
+import { AdminData, UserDetail } from './types'
+
+const getAdminEmails = (adminData: AdminData[]) => {
+  const emailArr: string[] = []
+  adminData?.forEach((data) => emailArr.push(data.email))
+  return emailArr
+}
```
Comment (oyo) [inline cx-portal/src/features/admin/userOwn/mapper.ts:27]: this function is not needed - see below

File: cx-portal/src/features/admin/userOwn/mapper.ts [hunk 2]
```diff
@@ -51,7 +57,10 @@ export const userDetailsToCards = (
         value: parsedToken ? parsedToken.organisation : 'N/A',
       },
       adminName: { label: 'Admin Name', value: 'N/A' },
-      adminMail: { label: 'Admin E-Mail', value: 'N/A' },
+      adminMail: {
+        label: 'Admin E-Mail',
+        value: user.admin?.length ? getAdminEmails(user.admin) : 'N/A',
```
Comment (oyo) [inline cx-portal/src/features/admin/userOwn/mapper.ts:62]: sure this value should be an array and not a string?
Anyway, better replace
`getAdminEmails(user.admin)`
with
`user.admin?.map((data: AdminData) => data.email)`

**Review by oyo — CHANGES_REQUESTED** [review]

File: cx-portal/src/features/admin/userOwn/mapper.ts (hunk 1 above)
Comment (nidhigarg-bmw) [reply cx-portal/src/features/admin/userOwn/mapper.ts]: @oyo Made the changes as requested

**Review by oyo — DISMISSED** [review]

**Review by oyo — APPROVED** [review]

**Pull request merged**